<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>性能分析</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <column>
         <property name="text">
          <string notr="true">标签</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>序号</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>历时(s)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>起始(ms)</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>计数(ms)</string>
         </property>
         <property name="background">
          <color>
           <red>255</red>
           <green>85</green>
           <blue>127</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>百分比</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数据1</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>数据2</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="PerfDisplay" name="view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>100</height>
         </size>
        </property>
        <property name="acceptDrops">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <zorder>treeWidget</zorder>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="baseSlider">
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>970</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="labelMenu">
    <property name="title">
     <string>标签</string>
    </property>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="labelMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PerfDisplay</class>
   <extends>QFrame</extends>
   <header>perfdisplay.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
